Readme.rst

```diff
@@ -18,13 +18,38 @@ When pip is ready you can do
 
    pip install djgentelella
 
+Install rest_framework
```

Comment:
Django Rest Framework is part of Djgentelella, and it's in requirements.txt so no extra step is required.